DahlDesignDDC/30_Switches.ino

```diff
@@ -27,4 +27,9 @@
   {
     processCommands();
   }
+#elif
```

Comment:
Might need to add `(BOARDTYPE == 2)` on this `#elif`, or we might have issues with other boards?

Reply:
Damn you’re right, I was too quick on that one. I fix that rapidly.